charts/tfy-notebook-controller/templates/deployment.yaml

```diff
@@ -18,6 +18,8 @@ spec:
         {{- with .Values.labels }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
You have to truncate it to 63 characters.

Reply:
Do we need to add quote here?

Reply:
> include (print $.Template.BasePath "/configmap.yaml")

Will this just take the template or will this take the rendered configmap yaml?

Reply:
it takes the rendered configmap yaml here